design/FY2017/NFP-Enhancement for Urban Microclimate.md

```diff
@@ -0,0 +1,99 @@
+	Zone,Average,Zone Outdoor Air Wetbulb Temperature [C]
+	Zone,Average,Zone Outdoor Air Wind Speed [m/s]
+
+For a simple co-simulation case, it would be straightforward to add new EMS actuators for these six variables to override their values directly. We propose to add the Zone Outdoor Air Wind Direction [degree] as a new report variable at every time step. This would enable the co-simulation of the interaction between buildings and the urban micro-climate.
+
+
+# Approaches 
+We propose to add the Zone Outdoor Air Wind Direction [degree] report variable at every time step, and make the seven variables as new EMS actuators that can be set up for overriding the environmental variable fields. This enables data exchange with the local ambient air conditions at arbitrary nodes or coordinates by setting up FMUs for co-simulation (Figure 2).
+
+![LocalNode](EnergyPlusLocalAirNode.png) 
+
+***Figure 2 Local ambient air conditions at zone and surface level***
+
+The local environmental conditions can be exchanged with or overwritten by external simulation or calculation results at each time step in run time.
+
+# Testing/Validation/Data Sources
+
+TBD
+
+# Input Output Reference Documentation
+
+TBD
+
+# Engineering Reference
+
+TBD
+
+# Output Details and Examples
+
+TBD
+
+# Example Files and Transition Changes
+
+No transition changes are envisioned as a result of the new features (but this can change based on feedback from the team). New example files will be developed to demonstrate the new features.
+
+# References
+
+n/a
+
+# E-mail comments
+
+**Lixing Gu <gu@fsec.ucf.edu> Thu, Jun 1, 2017** 
+
+LocalOutdoorAir:Node
+Since you require to input outdoor conditions, T, W, Wind Speed and direction, is it possible to add solar radiation treated as diffuse component?
+
+> That can be added in future using the same approach, but we don’t see this as an immediate need to support urban energy modeling. A separate topic is to allow the calculated solar radiation (diffuse, direct or combined) to be an input for external surfaces, which is outside the scope of this proposal.
+
+**Michael J Witte <mjwitte@gard.com> Mon, Jun 5, 2017** 
+
+Rather than add the concept of outdoor air nodes with one-to-one correspondence with surfaces and zones, here it may be better to just add new EMS actuators.  Surface and zone outdoor conditions are already tracked separately, and they are reportable:
+	Zone,Average,Surface Outside Face Outdoor Air Drybulb Temperature [C]
+	Zone,Average,Surface Outside Face Outdoor Air Wetbulb Temperature [C]
+	Zone,Average,Surface Outside Face Outdoor Air Wind Speed [m/s]
+	Zone,Average,Zone Outdoor Air Drybulb Temperature [C]
+	Zone,Average,Zone Outdoor Air Wetbulb Temperature [C]
+	Zone,Average,Zone Outdoor Air Wind Speed [m/s]
+
+So it would be straightforward to add new EMS actuators for these six variables to override these values directly.  The only thing missing is wind direction, which could be added to the same data structures.
```

Comment:
How will E+ use wind direction?  Don't you really want to know pressure at the surface so you can calculate infiltration?  Wouldn't pressure be available from a CFD calculation?